CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+```
+
+## Making a contribution
+
+1. Create a pull request and tag the Plotly team (`@plotly/dash`) and tag / request review from [@rmarren1](https://github.com/rmarren1) and [@T4rk1n](https://github.com/T4rk1n).
+2. After a review has been done and your changes have been approved, create a prerelease and comment in the PR. Version numbers should follow [semantic versioning][]. To create a prerelease:
+    * Add `rc1` to `version.py` (`./dash/version.py`) e.g. `0.13.0rc1`
+        - If needed, ask @chriddyp to get PyPi package publishing access.
+    * Run `python setup.py sdist` to build a distribution zip.
+    * Check the `dist` folder for a zip ending with your selected version number. Double check that this version number ends with `rc#`, as to not mistakenly publish the package.
+    * Run `twine upload dist/<package_name>`.
+3. Comment in the PR with the prerelease version
+4. Update the top-level comment to include info about how to install, a summary of the changes, and a simple example.
+    * This makes it easier for a community member to come in and try it out. As more folks review, it's harder to find the installation instructions deep in the PR
+    * Keep this top-level comment updated with installation instructions (e.g. the `pip install` command)
+5. Make a post in the [Dash Community Forum][]
+    * Title it `":mega: Announcement! New <Your Feature> - Feedback Welcome"`
+    * In the description, link to the PR and any relevant issue(s)
+    * Pin the topic so that it appears at the top of the forum for two weeks
+
+## [Checklists](http://rs.io/unreasonable-effectiveness-of-checklists/)
+### Pre-Merge checklist
+- [ ] All tests on CircleCI have passed.
+- [ ] All visual regression differences have been approved.
+- [ ] If changes are significant, a release candidate has been created and posted to Slack, the Plotly forums, and at the very top of the pull request.
+- [ ] You have updated the `dash/version.py` file and the top of `CHANGELOG.md`
+- [ ] Two people have :dancer:'d the pull request. You can be one of these people if you are a Dash core contributor.
+
+### Post-Merge checklist
+- [ ] You have tagged the release using `git tag v<version_number`.
+
+### Pre-Release checklist
+- [ ] Everything in the Pre-Merge checklist is completed. (Except the last two if this is a release candidate).
+- [ ] `git remote show origin` shows you are in the correct repository.
+- [ ] `git branch` shows that you are on the expected branch.
+- [ ] `git status` shows that there are no unexpected changes.
+- [ ] `dash/version.py` is at the correct version.
+
+### Post-Release checklist
+- [ ] You have closed all issues that this pull request solves, and commented the new version number users should install.
+
+## Financial Contributions
+
+If your company wishes to sponsor development of open source dash components, please [get in touch][].
```

Comment:
Let's say "Dash, and many of Plotly's open source products, have been funded through direct sponsorship by companies. [Get in touch] about funding feature additions, consulting, or custom app development."